CardanoSharp.Wallet/Extensions/Models/Transactions/TransactionOutputExtensions.cs

```diff
@@ -1,12 +1,16 @@
-﻿using CardanoSharp.Wallet.Models.Transactions;
-using PeterO.Cbor2;
+﻿
 using System;
 using System.Collections.Generic;
+using CardanoSharp.Wallet.Models.Transactions;
+using PeterO.Cbor2;
 
 namespace CardanoSharp.Wallet.Extensions.Models.Transactions
 {
 	public static partial class TransactionOutputExtensions
 	{
+		private static ulong adaOnlyMinUTxO = 1000000;
+		private static string dummyAddress = "addr_test1qzx9hu8j4ah3auytk0mwcupd69hpc52t0cw39a65ndrah86djs784u92a3m5w475w3w35tyd6v3qumkze80j8a6h5tuqq5xe8y";
```

Comment:
while i dont really care what address is used here, if you would prefer to use an address that we use in the tests

`addr_test1qpu5vlrf4xkxv2qpwngf6cjhtw542ayty80v8dyr49rf5ewvxwdrt70qlcpeeagscasafhffqsxy36t90ldv06wqrk2qum8x5w`

this is an address from the mnemonic we use in the TransactionTests.

Reply:
Yep sounds good I will update